core/lib/zksync_core/src/state_keeper/updates/miniblock_updates.rs

```diff
@@ -26,6 +26,7 @@ pub struct MiniblockUpdates {
     pub l1_gas_count: BlockGasCount,
     pub block_execution_metrics: ExecutionMetrics,
     pub txs_encoding_size: usize,
+    pub payload_encoding_size: usize,
```

Comment:
Future work (?): Could make sense to observe the payload size estimate as a histogram to have stats on it readily available. OTOH, at least for analytics it could be reconstructed later.

Reply:
What's the best way to check these kind of changes on local dev env?

Reply:
You can spin up Prometheus and Grafana locally, e.g. using `docker compose`, and check changes in Grafana. I think we have a config [on Notion](https://www.notion.so/matterlabs/Running-Grafana-Prometheus-locally-8770d8a67c4d43a8a65dd8fe2289ad09).